dashboard/app/models/foorm/submission.rb

```diff
@@ -11,5 +11,84 @@
 #
 
 class Foorm::Submission < ActiveRecord::Base
-  belongs_to :foorm_form
+  has_one :metadata, class_name: 'Pd::WorkshopSurveyFoormSubmission', foreign_key: :foorm_submission_id
```

Comment:
not every submission will have a pd submission--ex. the misc survey submissions (end of year teacher survey) have metadata in a different table. Will that cause issues here?

Reply:
I don't think this will cause problems if you tried to get `misc_survey_submission.metadata` (I think you'd just get nil), but we should probably add that as another `has_one` here as well (I'm kind of ignoring misc_survey for the moment...) I could rename this one `workshop_metadata`, and that one `misc_metadata` or something? I also don't know if there's a way to have a single `has_one` that works across multiple relations, which would be cool?

Reply:
Where is misc foorm metadata btw?

Reply:
https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/foorm/misc_survey.rb

Reply:
Ah ok, I saw this but thought it'd be named something like `misc_survey_foorm_submission`.